data/ecosystem.yaml

```diff
@@ -0,0 +1,217 @@
+collections:
+  name: Ansible collections
+  description: Ansible collections offer distributions of playbooks, roles, modules, and plugins.
+  resources:
+    index:
+      link: "https://docs.ansible.com/ansible/latest/collections/index.html"
+      text: Collection index
+    user_guide:
+      link: "https://docs.ansible.com/ansible/latest/collections_guide/index.html"
+      text: Find out how to use collections
+    contributor_guide:
+      link: "https://docs.ansible.com/ansible/devel/community/contributions_collections.html#collections-contributions"
+      text: Learn how to contribute to collections
```

Comment:
This is a total nit, but this one and the one above (the text) - are anomalies on this page. Everything else is 'Foo documentation' etc, but these two are more action-oriented.

What about 'Collection user guide' and 'Collection contributor guide'?

I don't feel strongly about this so just tossing the idea out there and will marke the PR as approved.

Reply:
I see what you mean but this is a faithful reproduction of the current ecosystem page on the docsite. If we want to change that, I'd suggest a follow up PR.

Reply:
ok cool. I can do a followon pr once this merges.